_posts/en/newsletters/2020-06-24-newsletter.md

```diff
@@ -0,0 +1,233 @@
+  the formula, its history, and even sample code from Bitcoin’s 0.1.5 release.
+
+- [Does Bitcoin really need timestamps?]({{bse}}96185)
+  Pieter Wuille explains why limiting the block rate without reference to a
+  clock time outside of the blockchain could make running full nodes more
+  expensive while also struggling to keep the stale block rate low and
+  preventing collusion attacks.
+
+- [In a fee overpayment attack, why can't compromised software provide fake previous transactions corresponding to fake inputs?]({{bse}}96309)
+  Regarding a [fee overpayment attack][news101 fee overpayment attack] on segwit
+  transactions with multiple inputs, justinmoon asks why the remedy of the
+  attack, requiring copies of previous transactions for the inputs, is not
+  vulnerable to malicious software providing fake previous transactions. Since
+  any provided previous transaction must have a hash that matches the spending input’s
+  previous transaction hash, such an attack is not feasible.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [LND 0.10.2-beta.rc2][lnd 0.10.2-beta] this release candidate for an
+  LND maintenance release is now available for testing.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #19260][] p2p: disconnect peers that send filterclear + update existing filter msg disconnect logic FIXME:dongcarl
+
+- [Bitcoin Core #19133][] rpc, cli, test: add bitcoin-cli -generate command FIXME:moneyball
+
+- [Bitcoin Core #18027][] adds two options to the GUI's *File* menu for
+  working with Partially-Signed Bitcoin Transactions ([PSBTs][topic
+  psbt]): *Load PSBT from file* and *Load PSBT from clipboard*.  When
+  one of those options is clicked and a PSBT is loaded, a dialog is
+  provided that allows the user to sign an incomplete PSBT if this
+  wallet has its key, broadcast a complete PSBT, or copy or save the
+  PSBT for processing with another tool such as [HWI][].  Combined with
+  other recent PSBT-related GUI changes (see Newsletters [#74][news74
+  psbtgui] and [#82][news82 psbtgui]) and [HWI's own GUI][hwi gui] this
+  makes it possible to use a PSBT-based process with Bitcoin Core for
+  the first time without using any RPCs.
+
+    ![Screenshot of PSBT dialog in Bitcoin Core GUI](/img/posts/2020-06-psbt-dialog.png)
+
+- [Bitcoin Core #16377][] updates the `walletcreatefundedpsbt` and
+  `fundrawtransaction` RPCs.  These RPCs normally use the wallet to
+  automatically choose which UTXOs to spend in a unsigned transaction,
```

Comment:
```suggestion
 choose which UTXOs to spend in an unsigned transaction automatically,
```

Reply:
What's the motivation for moving "automatically"?  (I'm not opposed, just curious.)

Reply:
It's a split infinitive.